### PR TITLE
Fix improve usability of file picker

### PR DESCRIPTION
**Pull request by Kiarokh**
fix: https://github.com/Lundalogik/crm-feature/issues/1064

This will improve the component for now, but this component should be totally re-designed!

## Review:
- [ ] Commits are [atomic](https://seesparkbox.com/foundry/atomic_commits_with_git)
- [ ] Commits have the correct *type* for the changes made
- [ ] Commits with *breaking changes* are marked as such

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:
- [ ] Chrome
- [ ] Edge
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

**Comment by Befkadu1** [comment]
I am not sure if the type of  this commit message `chore(file): align 'required' controls to bottom right .....` is chore

File: src/components/file/file.tsx [hunk 1]
```diff
@@ -121,6 +121,7 @@ export class File {
                 onDrop={this.handleFileDrop}
                 onDragEnter={this.preventAndStop}
                 onDragOver={this.preventAndStop}
+                leadingIcon={'upload_to_cloud'}
```
Comment (Befkadu1) [inline src/components/file/file.tsx:124]: We probably need to do it like that `leadingIcon="upload_to_cloud"`

**Review by Befkadu1 — CHANGES_REQUESTED** [review]

File: src/components/file/file.tsx (hunk 1 above)
Comment (adrianschmidt) [reply src/components/file/file.tsx:124]: Either ought to work I think. `leadingIcon="upload_to_cloud"` keeps the whole thing as html, whereas `leadingIcon={'upload_to_cloud'}` switches over to javascript for the value. The end result should be the same.

File: src/components/file/file.mdx [hunk 2]
```diff
@@ -6,8 +6,27 @@ menu: Components
 
 # File
 
+This component lets end-users select **only one** file from their device storage. Regardless of the user's device or operating system, this component opens up a file picker dialog that allows the user to choose a file.
```
Comment (adrianschmidt) [inline src/components/file/file.mdx:9]:
```suggestion
This component lets end-users select a *single* file from their device storage. Regardless of the user's device or operating system, this component opens up a file picker dialog that allows the user to choose a file.
```

File: src/components/file/file.mdx [hunk 3]
```diff
@@ -6,8 +6,27 @@ menu: Components
...
+
 <limel-props name="limel-file" />
 
+## Using correct labels
+
+This file picker can be used in different contexts. The component's distinct visual design including the upload icon hits end-users that this is not a normal input field like other fileds in the form for example.
```
Comment (adrianschmidt) [inline src/components/file/file.mdx]:
```suggestion
This file picker can be used in different contexts. The component's distinct visual design including the upload icon tells end-users that this is not a normal input field like other fields in the form for example.
```

File: src/components/file/file.mdx [hunk 4]
```diff
@@ -6,8 +6,27 @@ menu: Components
...
+
+However, you need to make sure whether you are using a descriptive `label` that clarifies the functionality of the file picker, and or provides users with clear instructions.
```
Comment (adrianschmidt) [inline src/components/file/file.mdx:17]:
```suggestion
However, you need to use a descriptive `label` that clarifies the functionality of the file picker, and/or provides users with clear instructions.
```

File: src/components/file/file.mdx [hunk 5]
```diff
@@ -6,8 +6,27 @@ menu: Components
...
+Depending on the context, you may need to avoid labels such as:
+- File
+- Document
+
+and instead consider using labels like:
+- Attach a file
+- Upload a file
+- Choose a document
+- Choose a file
+- and similar...
```
Comment (adrianschmidt) [inline src/components/file/file.mdx]:
```suggestion
- and similar…
```

File: src/components/file/file.mdx [hunk 6]
```diff
@@ -6,8 +6,27 @@ menu: Components
```
Comment (adrianschmidt) [inline src/components/file/file.mdx:6]: You don't have to change it this time, but "documentations" doesn't sound good. It's "documentation" in both singular and plural 🙂

**Comment by adrianschmidt** [comment]
> I am not sure if the type of this commit message `chore(file): align 'required' controls to bottom right .....` is chore

I agree. Since it fixes things in the docs, `docs` would be a better type than `chore`.

**Comment by Kiarokh** [comment]
tnx guys! I've pushed two fixups.I don't know how to change the commit message without doing a rebase and force push. Can you fix it?

**Comment by adrianschmidt** [comment]
> tnx guys! I've pushed two fixups.I don't know how to change the commit message without doing a rebase and force push. Can you fix it?

The only way to fix it is to do a rebase and force-push, so that's ok. Just avoid squashing the fixups while doing so. If you're unsure, I can do it for you (although practise makes perfect 😛)

**Comment by Befkadu1** [comment]
> > tnx guys! I've pushed two fixups.I don't know how to change the commit message without doing a rebase and force push. Can you fix it?
> 
> The only way to fix it is to do a rebase and force-push, so that's ok. Just avoid squashing the fixups while doing so. If you're unsure, I can do it for you (although practise makes perfect 😛)
> 
> `git rebase -i --no-autosquash master`

(although practise ..... )  `practice` 😛

**Comment by adrianschmidt** [comment]
> > > tnx guys! I've pushed two fixups.I don't know how to change the commit message without doing a rebase and force push. Can you fix it?
> > 
> > 
> > The only way to fix it is to do a rebase and force-push, so that's ok. Just avoid squashing the fixups while doing so. If you're unsure, I can do it for you (although practise makes perfect 😛)
> > `git rebase -i --no-autosquash master`
> 
> (although practise ..... ) `practice vs practise` 😛

Hmm… yeah, I guess it's the noun form that's used in that saying… true 👍

Practice makes perfect! 😆

**Comment by Kiarokh** [comment]
tnx Adrian. Please do it if you can. Right now I'm in a meeting (during which I'm multitasking as well :D ) But I don't wanna f*up things.

**Review by Befkadu1 — APPROVED** [review]

**Pull request merged**